packages/@webex/plugin-meetings/test/unit/spec/roap/turnDiscovery.ts

```diff
@@ -128,6 +130,42 @@ describe('TurnDiscovery', () => {
       assert.isUndefined(turnDiscoverySkippedReason);
     });
 
+    it('sends TURN_DISCOVERY_REQUEST when enable Multistream, waits for response and sends OK', async () => {
```

Comment:
this new test body is exactly the same as the existing test except for the first line. Can you extract it to a function? 
or even better would be to use parametrized tests

Reply:
done